testing/e2e/skr-tester/README.md

```diff
@@ -0,0 +1,5 @@
+# SKR Tester
+
+The SKR Tester is used for integration testing of the Kyma Environment Broker. 
```

Comment:
```suggestion
SKR Tester is used for integration testing of Kyma Environment Broker. 
```

Reply:
For proper names like Kyma Environment Broker, we don't use articles. "SKR Tester" seems to be a proper name as well, so I wouldn't use `the` with it.